packages/entities/README.md

```diff
@@ -0,0 +1,13 @@
+# @wordpress/entities
+
+Entities utils for WordPress.
```

Comment:
Can we at least say "keyboard entities" or something? 😄

Reply:
Oh geez, that makes more sense. I think I read the `TAB` import change at the top of the PR diff and my brain got confused. Time for more ☕️

"HTML Entities" would be great.